tests/phpunit/integration/Modules/Tag_ManagerTest.php

```diff
@@ -419,6 +493,63 @@ public function test_sanitize_container_name( $input, $expected ) {
 		);
 	}
 
+	/**
+	 * @dataProvider data_ads_measurement_data
+	 */
+	public function test_check_ads_measurement_connection( $tags, $expected_result ) {
+		$account_id            = '123456789';
+		$container_id          = 'GTM-987654321';
+		$internal_container_id = 'GTM-987654321';
```

Comment:
The internal ID is a numeric value. The container ID is what we call the container's `publicId` (GTM-prefixed value).